client/actions/auth.js

```diff
@@ -127,6 +141,20 @@ export function getUserDetails (userId) {
   }
 }
 
+export function getAllUsers () {
```

Comment:
This should have a unit test. Feel free to ask for my help if you need it (you might).